producer.py

```diff
@@ -72,28 +72,56 @@ async def main():
         # Consider only last two releases by default
         adviser_versions = [str(v) for v in source.get_sorted_package_versions(package_name)][:number_releases]
 
+    justifications_collected: List[Dict[str, Any]] = []
+
+    for adviser_version in adviser_versions:
+        justifications_collected = Adviser.create_adviser_dataframe(
+            adviser_version=adviser_version,
+            adviser_files=adviser_files,
+            justifications_collected=justifications_collected,
+        )
+
+    adviser_dataframe = Adviser._create_adviser_dataframe(justifications_collected)
+
+    if adviser_dataframe.empty:
+        _LOGGER.exception("No adviser documents identified")
+        return
+
+    adviser_dataframe["date_"] = [
+        pd.to_datetime(str(v_date)).strftime("%Y-%m-%d") for v_date in adviser_dataframe["date"].values
```

Comment:
isn't this already done in https://github.com/thoth-station/report-processing/blob/7cad3661eb629f4ad5b66c5e47f1368a2cc5ac65/thoth/report_processing/components/adviser.py#L210 ?

wouldn't this be sufficient?
```suggestion
        v_date.strftime("%Y-%m-%d") for v_date in adviser_dataframe["date"].values
```

Reply:
Thanks!

Reply:
Actually that is adjusted in new PR not yet merged, thanks!